runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemWithCharge.java

```diff
@@ -30,140 +30,15 @@
-import static net.runelite.api.ItemID.PHARAOHS_SCEPTRE_8;
-import static net.runelite.api.ItemID.RING_OF_DUELING1;
-import static net.runelite.api.ItemID.RING_OF_DUELING2;
-import static net.runelite.api.ItemID.RING_OF_DUELING3;
-import static net.runelite.api.ItemID.RING_OF_DUELING4;
-import static net.runelite.api.ItemID.RING_OF_DUELING5;
-import static net.runelite.api.ItemID.RING_OF_DUELING6;
-import static net.runelite.api.ItemID.RING_OF_DUELING7;
-import static net.runelite.api.ItemID.RING_OF_DUELING8;
-import static net.runelite.api.ItemID.RING_OF_RETURNING1;
-import static net.runelite.api.ItemID.RING_OF_RETURNING2;
-import static net.runelite.api.ItemID.RING_OF_RETURNING3;
-import static net.runelite.api.ItemID.RING_OF_RETURNING4;
-import static net.runelite.api.ItemID.RING_OF_RETURNING5;
-import static net.runelite.api.ItemID.RING_OF_WEALTH_1;
-import static net.runelite.api.ItemID.RING_OF_WEALTH_2;
-import static net.runelite.api.ItemID.RING_OF_WEALTH_3;
-import static net.runelite.api.ItemID.RING_OF_WEALTH_4;
-import static net.runelite.api.ItemID.RING_OF_WEALTH_5;
-import static net.runelite.api.ItemID.SKILLS_NECKLACE1;
-import static net.runelite.api.ItemID.SKILLS_NECKLACE2;
-import static net.runelite.api.ItemID.SKILLS_NECKLACE3;
-import static net.runelite.api.ItemID.SKILLS_NECKLACE4;
-import static net.runelite.api.ItemID.SKILLS_NECKLACE5;
-import static net.runelite.api.ItemID.SKILLS_NECKLACE6;
-import static net.runelite.api.ItemID.SLAYER_RING_1;
-import static net.runelite.api.ItemID.SLAYER_RING_2;
-import static net.runelite.api.ItemID.SLAYER_RING_3;
-import static net.runelite.api.ItemID.SLAYER_RING_4;
-import static net.runelite.api.ItemID.SLAYER_RING_5;
-import static net.runelite.api.ItemID.SLAYER_RING_6;
-import static net.runelite.api.ItemID.SLAYER_RING_7;
-import static net.runelite.api.ItemID.SLAYER_RING_8;
-import static net.runelite.api.ItemID.TELEPORT_CRYSTAL_1;
-import static net.runelite.api.ItemID.TELEPORT_CRYSTAL_2;
-import static net.runelite.api.ItemID.TELEPORT_CRYSTAL_3;
-import static net.runelite.api.ItemID.TELEPORT_CRYSTAL_4;
-import static net.runelite.api.ItemID.TELEPORT_CRYSTAL_5;
-import static net.runelite.api.ItemID.WATERING_CAN;
-import static net.runelite.api.ItemID.WATERING_CAN1;
-import static net.runelite.api.ItemID.WATERING_CAN2;
-import static net.runelite.api.ItemID.WATERING_CAN3;
-import static net.runelite.api.ItemID.WATERING_CAN4;
-import static net.runelite.api.ItemID.WATERING_CAN5;
-import static net.runelite.api.ItemID.WATERING_CAN6;
-import static net.runelite.api.ItemID.WATERING_CAN7;
-import static net.runelite.api.ItemID.WATERING_CAN8;
-import static net.runelite.api.ItemID.WATERSKIN0;
-import static net.runelite.api.ItemID.WATERSKIN1;
-import static net.runelite.api.ItemID.WATERSKIN2;
-import static net.runelite.api.ItemID.WATERSKIN3;
-import static net.runelite.api.ItemID.WATERSKIN4;
+import static net.runelite.api.ItemID.*;
```

Comment:
You should avoid wildcard imports. Make sure to configure your IDE to not do this in the future.

Reply:
> 
> 
> You should avoid wildcard imports. Make sure to configure your IDE to not do this in the future.

Wildcard is accepted for static imports iirc.

Reply:
It certainly isn't upstream. And everywhere else it isn't done like that. The project configuration by default won't allow it unless its 99+ imports.

Reply:
https://github.com/runelite-extended/runelite/wiki/Code-Conventions

```By changing Names count to use static imports with '*' to 10 (static imports for f.e. ItemIDs with wildcards are preferred)```